src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -68,9 +68,16 @@ protected function describeRoute(Route $route, array $options = []): void
 
     protected function describeContainerParameters(ParameterBag $parameters, array $options = []): void
     {
+        $deprecatedParameters = $parameters->allDeprecated();
+
         $this->write("Container parameters\n====================\n");
         foreach ($this->sortParameters($parameters) as $key => $value) {
-            $this->write(sprintf("\n- `%s`: `%s`", $key, $this->formatParameter($value)));
+            $this->write(sprintf(
+                "\n- `%s`: `%s`%s",
+                $key,
+                $this->formatParameter($value),
+                isset($deprecatedParameters[$key]) ? sprintf(' *(Deprecated since %s %s: %s)*', $deprecatedParameters[$key][0], $deprecatedParameters[$key][1], sprintf(...\array_slice($deprecatedParameters[$key], 2))) : ''
```

Comment:
```suggestion
                isset($deprecatedParameters[$key]) ? sprintf(' *(Since %s %s: %s)*', $deprecatedParameters[$key][0], $deprecatedParameters[$key][1], sprintf(...\array_slice($deprecatedParameters[$key], 2))) : ''
```